website/templates/project/retracted_registration.mako

```diff
@@ -0,0 +1,65 @@
+<div id="projectScope">
```

Comment:
@sloria I agree that there is. This was the simplest solution I could find after trying several alternatives. I think that the project templates need a solid reorganization as a whole. I'd prefer to not do that in this PR though if that's alright.